pollination_apps/cli/__init__.py

```diff
@@ -196,10 +199,15 @@ def run(path, owner, name, tag):
 
     if tag is None:
         tag = 'latest'
+    
+    # optionally mount the current path as a volume to the container
+    volume = ''
+    if editable:
+      volume = f'-v {path}/:/app'
```

Comment:
You can make this look more pythonic if you like.

```suggestion
    volume = f'-v {path}/:/app' if editable else ''
```

In any case, we use four spaces for indentation rather than two.